enterprise/migrations/0038_auto_20180122_1427.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-01-22 20:27
+from __future__ import unicode_literals
+
+import fernet_fields.fields
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0037_auto_20180110_0450'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enterprisecustomerreportingconfiguration',
+            name='decrypted_password',
```

Comment:
I worry a little naming these `decrypted` will confuse people when they are in fact encrypted at rest.  Although describing it as decrypted in the model will caution people from displaying it - is that the intent?

Reply:
Yeah, I went back and forth on this as well. I think this is the right name.